jalib/jfilesystem.cpp

```diff
@@ -229,9 +235,12 @@ bool jalib::Filesystem::FileExists ( const jalib::string& str )
 
 jalib::StringVector jalib::Filesystem::GetProgramArgs()
 {
-  static StringVector rv;
+  static StringVector *rv = NULL;
+  if (rv == NULL) {
```

Comment:
Suggest adding comment:
// Technically, this is a memory leak, but `rv` is static and so it happens only once.